src/Adapter/Configuration.php

```diff
@@ -255,9 +255,9 @@ private function hasMultilang(string $key, ?int $shopId, ?int $shopGroupId, bool
      *
      * @param string $key
      *
-     * @return self
+     * @return $this
```

Comment:
```suggestion
     * @return self
```

Prefers self for inheritance